tests/services/test_service_users.py

```diff
@@ -12,6 +12,20 @@
 from invenio_access.permissions import system_identity
 from invenio_records_resources.services.errors import PermissionDeniedError
 
+from invenio_users_resources.proxies import current_actions_registry
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
when testing the user service, always set the after actions to nothing to mitigate side effects from other modules.